demos/object_detection_demo/jupyter-python/requirements.txt

```diff
@@ -0,0 +1,10 @@
+requests
+jupyterlab
+ipywidgets
+voila
+matplotlib
+openvino
+opencv-python-headless
```

Comment:
Why are these included here? We should be using the OpenVINO and OpenCV libraries from the OpenVINO toolkit package we're in, not from PyPI.

Reply:
We wanted these notebooks to work as standalone demo's, with instructions that work for people who are completely new to OpenVINO. I can remove openvino from the requirements.txt and explain say in the README to install OpenVINO first. OpenCV is not installed by the PyPI distribution of OpenVINO (as far as I know), so I think that should remain in the requirements for the notebook.

Reply:
> We wanted these notebooks to work as standalone demo's, with instructions that work for people who are completely new to OpenVINO.

I can understand that, but if you're including them in OMZ, they are de facto not standalone demos - they are a part of OMZ, which is in itself part of the OpenVINO toolkit. It doesn't make sense to install OpenVINO with pip if you found this demo inside the OpenVINO toolkit.

It's not unreasonable to support the standalone scenario, but it should be understood that it's not the only possibility.

> I can remove openvino from the requirements.txt and explain say in the README to install OpenVINO first.

That would be reasonable.

> OpenCV is not installed by the PyPI distribution of OpenVINO (as far as I know), so I think that should remain in the requirements for the notebook.

OpenCV is in the same situation as OpenVINO - it only needs to be installed if we're not in the OpenVINO toolkit.

Reply:
@dkurt or @mryzhov Installing OpenVINO with `pip install openvino` or `pip install openvino-dev` from the testing wheels did not install OpenCV. Has that changed and is there a way to get an OpenVINO environment with OpenCV installed with the pip packages?

Reply:
I tested the very last openvino-dev package and saw that opencv is installed there, so I can remove it from the requirements of the notebook. @mryzhov could you confirm if opencv will always be included in openvino-dev? I am worried that if I assume requirements of other packages, the other packages may change and then my code breaks. But since OMZ is part of OpenVINO, if we know this will always be installed that is not a concern in this case.

Reply:
Yes, the openvino-dev wheel requires opencv as dependence